src/engine/Actor.ts

```diff
@@ -562,6 +564,25 @@ module ex {
           }
        }
     }
+    /**
+     * Gets the z-index of an actor. The z-index determines the relative order an actor is drawn in.
+     * Actors with a higher z-index are drawn on top of actors with a lower z-index
+     */
+    public getZIndex(): number {
+       return this._zIndex;
+    }
+
+    /**
+     * Sets the z-index of an actor and updates it in the drawing list for the scene. 
+     * The z-index determines the relative order an actor is drawn in.
+     * Actors with a higher z-index are drawn on top of actors with a lower z-index
+     * @param actor The child actor to remove
+     */
+    public setZIndex(newIndex: number) {
+       this.scene._sortedDrawingTree.remove(this);
+       this._zIndex = newIndex;
+       this.scene._sortedDrawingTree.add(this);
```

Comment:
To do that we'd need to make the z-index property accessible from outside the Actor, and that could lead to a scenario where the z-index has been updated but the change is not reflected in the scene's sorted drawing tree.